server/src/main/java/org/opensearch/indices/recovery/ForceSyncRequest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.indices.recovery;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.index.shard.ShardId;
+
+import java.io.IOException;
+
+/**
+ * Request to force a round of segment replication on primary target
+ *
+ * @opensearch.internal
+ */
+public class ForceSyncRequest extends RecoveryTransportRequest {
```

Comment:
How long can the force sync take with remote store and bigger segments? Do we need wait_for params?

Reply:
Thank you @Bukhtawar for raising this point. 
During `phase1` of recovery process, the target is synced with the source node. This action to force segment replication step (before finalize recovery) ensures new segments (if any) sync resulted from auto refreshes on older primary post phase 1 and thus should be light weight. Based on this, I think `wait_for` may not be needed. WDYT ?